src/LockTables/Request.php

```diff
@@ -11,11 +11,10 @@
 namespace Manticoresearch\Buddy\LockTables;
 
 use Manticoresearch\Buddy\Base\CommandRequestBase;
-use Manticoresearch\Buddy\Enum\ManticoreEndpoint;
 use Manticoresearch\Buddy\Network\Request as NetRequest;
 
 final class Request extends CommandRequestBase {
-	public ManticoreEndpoint $endpoint;
+	public string $endpoint;
```

Comment:
This change is caused by elastic-like requests too.  Since they have query paths like: `/<index_name>/_create/<doc_id>` , we cannot use the mapping to the endpoint Enum for them.

Reply:
Well, I guess my suggestion here in that case is better to rename to the path that is more native, not endpoint. What do you think?

Reply:
Ok, let's do this.